content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -5,5 +5,223 @@ weight: 2
+
+{{< image width="100%"
+link="destinationonly.svg"
+caption="Waypoint proxies"
+>}}
+
+Destination-only waypoint simplifies the configuration process by eliminating the need for sidecar proxies and `exportTo` configurations. Waypoint proxies only need to be aware of the endpoints, pods, and workloads within their respective namespaces or service accounts. This streamlined approach reduces the complexity of L7 management and enables a more efficient use of resources.
+
+{{< image width="100%"
+link="destination-only-waypoint.svg"
+caption="Waypoint proxies"
+>}}
+
+* **Policy Enforcement**: In traditional Istio deployments, both source-side and destination-side policies were employed, which often led to confusion for users regarding policy enforcement and troubleshooting. Destination-only waypoint simplifies this process by enforcing all policies exclusively at the destination workload's namespace or service account. This centralized approach eliminates the need to track policies across multiple locations, making it easier to understand, manage, and troubleshoot L7 security configurations.
+
+{{< image width="100%"
+link="policies-enforced (1).svg"
+caption="Policy Enforced on Destination waypoint"
+>}}
+
+* **Mixed Environment**: In a mixed environment where clients may reside inside or outside the Istio mesh, destination-only waypoint ensures consistent policy enforcement regardless of the client's location. Since all policies are applied at the destination workload, users can be confident that security measures are consistently applied to all incoming traffic.
+
+{{< image width="100%"
+link="mixed-environment.svg"
+caption="Waypoint proxies"
+>}}
+
+### Handling destinations without waypoint proxies
+
+While destination-only waypoint offers centralized policy enforcement and simplified configuration, there may be instances where the destination workload doesn't have a waypoint proxy deployed. This could arise when connecting to external services beyond the control of the Istio mesh.
+
+To address this scenario, the Istio community is actively developing mechanisms to route traffic to the egress gateway and enable policy enforcement for destinations without waypoint proxies. This functionality will allow users to configure resilience-enhancing policies, such as timeouts, for external services.
+
+Please stay tuned for future blog posts and documentation updates that will provide detailed information on this evolving feature.
+
+## Deploying an application
+
+When someone with Istio admin privileges sets up Istio mesh, it becomes available for all users in specific namespaces. The examples below shows how Istio can be used transparently once it's successfully deployed in ambient mode and the namespaces are annotated accordingly.
+
+### Basic application deployment without ambient
+
+This section is Under Construction.
```

Comment:
This would a be a great thing to show- you can deploy the example apps, _then_ install Istio in ambient mode without having to restart the applications. You can probably use the sleep/notsleep example apps from Lin's getting started guide: https://istio.io/latest/blog/2022/get-started-ambient/#deploy-your-applications. 

If you wanted this section to show how you can have sidecars injected in an ambient-labeled namespace: 

```
kubectl label namespace default istio.io/dataplane-mode=ambient --overwrite=true
kubectl apply -f sleep/sleep.yaml -n default

# notsleep is in sidecar mode
kubectl apply -f <(istioctl kube-inject -f sleep/notsleep.yaml) -n default
```